services/movie.py

```diff
@@ -0,0 +1,30 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list = None, actors_ids: list = None) -> QuerySet | Movie:
+    movies = Movie.objects.all()
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids)
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    movie = Movie.objects.get(id=movie_id)
+    return movie
+
+
+def create_movie(movie_title: str,
+                 movie_description: str,
+                 genres_ids: list = [],
+                 actors_ids: list = []) -> Movie:
```

Comment:
It is very dangerous to use [] as a default. Use None insted and specify types like in example above.